pkg/util/util.go

```diff
@@ -139,11 +139,18 @@ func ConvertToIBytes(value string) string {
 	if value == "" {
 		return value
 	}
-	iBytes, err := units.RAMInBytes(value)
+	var bytes int64
+	var err error
+	if strings.Contains(value, "i") {
+		bytes, err = units.RAMInBytes(value)
+	} else {
+		bytes, err = units.FromHumanSize(value)
+	}
+	// should error be also returned?
```

Comment:
maybe returning an error is too much of an ask, not super super helpful because then caller would likely start swallowing the error

Reply:
Returning error would bring a lot of changes as we are calling this directly in the metav1.TableRow.